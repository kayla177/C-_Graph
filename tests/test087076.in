ENTITY H4QAO86U name_H4QA conference
ENTITY AYIO4M8F name_AYIO journal
ENTITY TNX1IGIV name_TNX1 institution
RELATIONSHIP H4QAO86U presented_at AYIO4M8F 9.5
PRINT H4QAO86U
HIGHEST
EXIT
