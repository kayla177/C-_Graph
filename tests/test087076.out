success
success
success
success
AYIO4M8F
H4QAO86U AYIO4M8F 9.5
